fidesctl/src/fidesapi/sql_models.py

```diff
@@ -1,3 +1,5 @@
+# type: ignore
```

Comment:
Why is this needed to fix test failures?

Reply:
mypy is unhappy about some of the type magic going on here, this was already being ignored but at the config level. I moved it here for transparency